package.json

```diff
@@ -17,7 +17,7 @@
         "dev": "encore dev",
         "watch": "encore dev --watch",
         "build": "encore production",
-        "pretty": "prettier --write \"./assets/*.{js,css,scss}\""
+        "pretty": "prettier --write \"./assets/**/*.{js,css,scss}\""
```

Comment:
If prettyCI don't work, I think it('s because of this, the folder is not right
I will now create a suggestion that I will commit to check if this fix PrettyCI on the build